internal/proto/local/controller/storage/host/static/store/v1/static.proto

```diff
@@ -105,6 +105,11 @@ message HostSet {
   // version allows optimistic locking of the resource
   // @inject_tag: `gorm:"default:null"`
   uint32 version = 7;
+
+  // This is not used! This is here to satisfy mask manager requirements due to
+  // shared API proto.
+  // @inject_tag: `gorm:"-"`
+  int32 sync_interval_seconds = 22 [(custom_options.v1.mask_mapping) = {this: "SyncIntervalSeconds" that: "sync_interval_seconds"}];
```

Comment:
Consider using the approach I used [here](https://github.com/hashicorp/boundary/pull/1684/files#diff-11ec4c832d135259149a5225ad14f972473201078b9f058b9ab31fa6b9ff67abR124) and [here](https://github.com/hashicorp/boundary/pull/1684/files#diff-38ab747f010825b0e56d6afa5b7b6cb55af54f24706c918f170120e36e74b1d7R68) which pulls the fields which are not relevant to the type into their own proto message.

Reply:
Great idea, thank you!